cmd/kanikoExecute.go

```diff
@@ -74,11 +109,51 @@ func runKanikoExecute(config *kanikoExecuteOptions, telemetryData *telemetry.Cus
 				log.SetErrorCategory(log.ErrorConfiguration)
 				return errors.Wrapf(err, "failed to read registry url %v", config.ContainerRegistryURL)
 			}
-			containerImageTag := fmt.Sprintf("%v:%v", config.ContainerImageName, strings.ReplaceAll(config.ContainerImageTag, "+", "-"))
-			dest = []string{"--destination", fmt.Sprintf("%v/%v", containerRegistry, containerImageTag)}
+
 			commonPipelineEnvironment.container.registryURL = config.ContainerRegistryURL
-			commonPipelineEnvironment.container.imageNameTag = containerImageTag
+
+			// Docker image tags don't allow plus signs in tags, thus replacing with dash
+			containerImageTag := strings.ReplaceAll(config.ContainerImageTag, "+", "-")
+
+			if config.ContainerMultiImageBuild {
+				log.Entry().Debugf("Multi-image build activated for image name '%v'", config.ContainerImageName)
+				i, err := docker.ImageListWithFilePath(config.ContainerImageName, config.ContainerMultiImageBuildExcludes, fileUtils)
```

Comment:
`i` is very short. Maybe `imageList` or even `imageListWithFilePath`?

Reply:
good point, will change this